### DISCUSSION TITLE
Feeds should be never-ending like on other platforms

### DISCUSSION THREAD

**Discussion by kingmigdor** [community/community#19910] [Other Feature Feedback, Questions, & Ideas]
I think it would be bad to have a feed that ends after like x amount of stuff if you are really looking for inspiration or something without knowing what you are looking for. For you's on other social media platforms have infinity for you pages which keeps the users on the platform, and for some inspiration stuff it should not be ending at some point.

**Comment by IlaySkutelsky** [comment]
Absolutely right! The feed used to be never ending and once a week or two I would scroll and get updates from various projects I'd never hear of any other way. Now the feed just stops randomly. Please change this back

**Comment by jfullstackdev** [comment]
but others are saying the opposite, they should avoid distractions from projects they don't know, 

as for me, just like feeds in other social networks, it should be really random and equal opportunity, so that each of us has the opportunity to see other works and the chance of that to be discovered, same applies to our projects

**Reply by kingmigdor** [reply]
Maybe projects should have a setting to become discoverable, setting up some tags for it and stuff

**Comment by AyushSehrawat** [comment]
Here i am only seeing a certain guy activities as feeds for last few weeks ;-; and some other stuff...

**Comment by jfullstackdev** [comment]
hello guys, there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)
